cirq-core/cirq/sim/act_on_state_vector_args.py

```diff
@@ -39,7 +39,9 @@ class ActOnStateVectorArgs(ActOnArgs):
 
     def __init__(
         self,
-        target_tensor: np.ndarray,
+        target_tensor: Optional[np.ndarray] = None,
+        initial_state: 'cirq.STATE_VECTOR_LIKE' = 0,
```

Comment:
Same, was wanting the `target_tensor` type to be extended rather than adding a new type. And dtype should be moved to the last param.

Reply:
Thanks, you can resolve this one.